std/experimental/logger/core.d

```diff
@@ -1763,7 +1763,7 @@ private @property Logger stdThreadLocalLogImpl() @trusted
 {
     import std.conv : emplace;
 
-    static ubyte[__traits(classInstanceSize, StdForwardLogger)] _buffer;
+    static /*align(StdForwardLogger.alignof)*/ void[__traits(classInstanceSize, StdForwardLogger)] _buffer;
```

Comment:
> Could work with size_t replaced by void*

At first I though about proposing `void*` but then I chose `size_t` for simplicity. But shouldn't all integral types N have `N.alignof == N.sizeof`?

> It get's pretty verbose, though.

Agreed, that's why I'm proposing to abstract those details in a `singleton(T)` template ;)